addons/popochiu/engine/objects/graphic_interface/graphic_interface.tscn

```diff
@@ -45,6 +45,7 @@ offset_left = -8.0
 offset_top = -8.0
 offset_right = 8.0
 offset_bottom = 8.0
+grow_horizontal = 2
```

Comment:
What is this for?

Reply:
To define how a Control node should grow. From Godot docs:

> The control will grow in both directions equally to make up if its minimum size is changed to be greater than its current size.

Reply:
Ah, yeah, I'm dumb! Sorry